src/utilities/theme/tokens.ts

```diff
@@ -4,12 +4,13 @@ const BorderRadius = {
 };
 
 const Shadow = {
+  shadowFromDimLight: 'rgba(0, 0, 0, 0.2)',
```

Comment:
You'll want to add this to tokens instead: https://github.com/Shopify/polaris-tokens/blob/next/formats/utils/color-factory/configs/base.js#L98

Reply:
Can we get into the habit of adding these locally until design reviews are completed and then moving them to tokens?

It just seems like a round about way of doing things. I'd have to add the token in the package, bump the version and then make the change in Polaris.

I'd prefer adding locally and then removing chunks later on as they become available in tokens

Reply:
I'm comfortable with that. But it should have an accompanying issue for the removal here, and a pull request in tokens.

Reply:
I'll put the PR up in tokens. Shouldn't we have a PR to move a bunch of local tokens? Or is this the only one you think worth moving right now?

Reply:
Currently the only tokens that live in polaris tokens are those generated by the color factory, so no, there shouldn't be anything else we need to migrate currently from `tokens.ts`

Reply:
As we start to explore customizable tokens for font-size, border-radius, we'll work to move those tokens over, but that work is not ready yet.